### PR TITLE
Completed css-positioning.

### PR DESCRIPTION
**Pull request by randygerada**
Screen Shot 2022-10-14 at 9 54 40 AM
Screen Shot 2022-10-14 at 9 54 10 AM

### CODE REVIEW

File: css-positioning/absolute.css [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  color: white;
+  -webkit-text-stroke: 1px black;
+  text-transform: uppercase;
+
+  /* your code here */
+  position: absolute;
+  bottom: 1px;
+}
+
+.overlay {
```
Comment (BrettAlbright) [inline css-positioning/absolute.css:38]: this element is not showing up in the example picture you submitted. Make sure to inspect it and confirm that its large enough to cover the whole element

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
small update needed

**Comment by randygerada** [comment]
Screen Shot 2022-10-14 at 12 00 56 PM

**Review by Zhiwutian — APPROVED** [review]
👍🏼

**Pull request merged**